tck/running_the_tck.asciidoc

```diff
@@ -58,13 +58,13 @@ To enable the tests in your project you need to add the following dependency to
     <dependency>
       <groupId>org.eclipse.microprofile.health</groupId>
       <artifactId>microprofile-health-tck</artifactId>
-      <version>1.0-SNAPSHOT</version>
+      <version>2.0.2</version>
```

Comment:
Thanks for this catch. However, it is not clear that the next version will be 2.0.2 (and it probably won't be needed). Can you rather use "2.1-SNAPSHOT", please?

Reply:
done